lib/api.js

```diff
@@ -68,6 +72,17 @@ function parseApiResponse(respData, callback) {
   callback(null, respData.result);
 }
 
+function pushOntoRetryQueue() {
+  retryQueue.push(arguments);
+
+  if (!retryHandle) {
+    retryHandle = setInterval(function() {
```

Comment:
You are right, just leaving it simple as it will be a no-op is the better choice.